schemas/user.json

```diff
@@ -43,6 +47,21 @@
 				"date:Y-m-d H:i:s"
 			]
 		},
+		"dateLastEmail": {
+			"type": "string",
+			"validation": [
+				"date:Y-m-d H:i:s"
+			]
+		},
+		"dateLastLogin": {
+			"type": "string",
+			"validation": [
+				"date:Y-m-d H:i:s"
+			]
+		},
+		"inlineHelp": {
+			"type": "boolean"
+		},
```

Comment:
It's up to you how you handle this, but one quirk of the API/validation stuff when editing an object is that you need to permit something to be `nullable` in order to support resetting it. For example, if someone had a setting of `inlineHelp: true` and you wanted to remove this (without setting it to false), you have to pass a `null` value to the API. That will throw a validation error unless you add this validation rule:

```
"validation": [
    "nullable"
]
```

Reply:
This is one of those areas I'm a little less confident about. Based on this, I would guess most non-required properties should be nullable, e.g. `affiliation`, `billingAddress`, `familyName`, maybe even `interests`... -- do I have that right?